lib/gears/shape.lua

```diff
@@ -751,58 +751,134 @@ end
 -- @tparam number h The shape height
 -- @tparam number percent The progressbar percent
 -- @tparam boolean hide_left Do not draw the left side of the shape
+-- @tparam number radius The arc radius
```

Comment:
First of all, thanks for reviewing code!
So I added `opt=...` for `percent`, `hide_left` and `radius` options but for `radius` option, I noticed few lines before (in arc function and also in circle function) that: `-- @tparam[opt=math.min(width height)/2] number thickness The arc thickness`. So that I took this syntax without comma, is it correct?

Reply:
Sorry for the delayed reply...

I'm not sure to understand what you mean by "this syntax without comma" but I'm guessing it's correct. Just push your changes :)